### ISSUE TITLE
Mobile IE scroll problem

### ISSUE THREAD

**Issue by no149** [zhukov/webogram#1049]
Hello

When viewing the app in mobile IE , I can't scroll up or down the chats view. Whenever I try that, the chat item beneath my finger opens up instead.

I'd like to contribute to solve this issue. I am also willing contribute code that takes advantage of any other mobile IE feature like live tiles etc. :)

**Comment by Ryuno-Ki** [comment]
I used [Real Favicon Generator](https://realfavicongenerator.net/) to create a _browserconfig.xml_ and adapted it for a project of mine.

Live tiles are basically a RSS feed with some editional meta data (colour of tile and so on), isn't it?

**Comment by zhukov** [comment]
@no149 go ahead. I will definitely merge changes (if you don't break other browsers of course :smile: )

**Comment by zhukov** [comment]
Closing the issue for now. @no149 feel free to open when you have any news.

**Comment by no149** [comment]
Hello!

I have found the code that caused the scroll problem:
`<a class="im_dialog" ng-mousedown="dialogSelect(dialogMessage.peerString, dialogMessage.unreadCount == -1 && dialogMessage.mid)">`
link to file: https://github.com/zhukov/webogram/blob/master/app/partials/mobile/dialog.html

The ng-mousedown breaks the scrolling in IE11 mobile. Changing it to 'click' solves the issue. I tested the changed version in Chrome as well and it was OK too.

But note that the problem doesn't happen in Edge. So this is not an important issue at all, much less, when you consider the  percent of the Windows Phone users who choose to use to web app instead of the official Windows client. But it annoyed me as one of the few such people. 

So shall I go on with fixing this? As said in the OP, I also would like to help in incorporating Edge/Windows 10 specific features into the client.
